chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -459,6 +459,70 @@ public ActivityDetails activityById(@PathVariable("id") final Long id)
                 id, startDate, endDate);
     }
 
+    @ApiOperation(value = "Get metadata about auditable records in the system for users.",
+            notes = "Users must specify 'start' and 'end' parameters to restrict the date range of the results.")
+    @RequestMapping(value = "/metadata/users", method = RequestMethod.GET,
+    produces = "application/json; charset=utf-8")
+    public List<ActivityMetadata> metadataForUsers(@RequestParam final Long start,
+            @RequestParam final Long end) throws JsonParseException, IOException, ValidationException {
+        Date startDate = new Date(start);
+        Date endDate = new Date(end);
+        validateActivityDatesAndDateRange(start, end);
+        return activityMetadataManager.getUserMaintenanceActivityMetadata(startDate, endDate);
+    }
+    
+    @ApiOperation(value = "Get metadata about auditable records in the system for announcements.",
+            notes = "Users must specify 'start' and 'end' parameters to restrict the date range of the results.")
+    @RequestMapping(value = "/metadata/announcements", method = RequestMethod.GET,
+    produces = "application/json; charset=utf-8")
+    public List<ActivityMetadata> metadataForAnnouncements(@RequestParam final Long start,
+            @RequestParam final Long end) throws JsonParseException, IOException, ValidationException {
+        Date startDate = new Date(start);
+        Date endDate = new Date(end);
+        validateActivityDatesAndDateRange(start, end);
+        return activityMetadataManager.getActivityMetadataByConcept(
+                ActivityConcept.ANNOUNCEMENT, startDate, endDate);
+    }
+    
+    @ApiOperation(value = "Get metadata about auditable records in the system for pending listings.",
+            notes = "Users must specify 'start' and 'end' parameters to restrict the date range of the results.")
+    @RequestMapping(value = "/metadata/pending_listings", method = RequestMethod.GET,
+    produces = "application/json; charset=utf-8")
+    public List<ActivityMetadata> metadataForPendingListings(@RequestParam final Long start,
+            @RequestParam final Long end) throws JsonParseException, IOException, ValidationException {
+        Date startDate = new Date(start);
+        Date endDate = new Date(end);
+        validateActivityDatesAndDateRange(start, end);
+        return activityMetadataManager.getActivityMetadataByConcept(
```

Comment:
Same comment about permissions. ONC and ADMIN should be able to see all pending listing activity and ACBs should see activity for their own pending listings and no one else should see any.